### ISSUE TITLE
Documentation inconsistency for minion ping_interval timing

### ISSUE THREAD

**Issue by cruscio** [saltstack/salt#44734]
### Description of Issue/Question

The documentation for minion ping_interval indicates the value is defined in seconds, but it's applied as minutes in minion.py

#### Documentation
https://docs.saltstack.com/en/latest/ref/configuration/minion.html#ping-interval
> Instructs the minion to ping its master(s) every n number of seconds. Used primarily as a mitigation technique against minion disconnects

#### Code
https://github.com/saltstack/salt/blob/2017.7.2/salt/minion.py#L2207
```
# schedule the stuff that runs every interval
ping_interval = self.opts.get(u'ping_interval', 0) * 60
if ping_interval > 0 and self.connected:
  def ping_master():
    try:
      def ping_timeout_handler(*_):
        if not self.opts.get(u'auth_safemode', True):
          log.error(u'** Master Ping failed. Attempting to restart minion**')
          delay = self.opts.get(u'random_reauth_delay', 5)
          log.info(u'delaying random_reauth_delay %ss', delay)
          # regular sys.exit raises an exception -- which isn't sufficient in a thread
          os._exit(salt.defaults.exitcodes.SALT_KEEPALIVE)

      self._fire_master('ping', 'minion_ping', sync=False, timeout_handler=ping_timeout_handler)
    except Exception:
      log.warning(u'Attempt to ping master failed.', exc_on_loglevel=logging.DEBUG)
  self.periodic_callbacks[u'ping'] = tornado.ioloop.PeriodicCallback(ping_master, ping_interval * 1000, io_loop=self.io_loop)
```
Looks like the code should be changed to not multiply ping_interval by 60

**Comment by thatch45** [comment]
Change the docs to match the code, this code is 3 years old. No reason to pull the rug out from under people because we had a bad doc.

**Comment by gtmanfred** [comment]
@cruscio would you mind submitting a PR to correct this doc?

Thanks,
Daniel

**Comment by rallytime** [comment]
Closed via #44770
